README.md

```diff
@@ -13,48 +14,60 @@ ElectionGuard is always improving. To keep up with the latest, check our **[offi
 
 ### Documentation
 
-This repository is a living document to help everyone interact with ElectionGuard. The [official ElectionGuard site](https://microsoft.github.io/electionguard) is built using the `/docs` folder and [mkdocs](https://www.mkdocs.org/) with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/). Ensure you have the python 3.8 or newer installed and run `make` to install the dependencies and start the site. 
+This repository is a living document to help everyone interact with ElectionGuard. The [official ElectionGuard site](https://microsoft.github.io/electionguard) is built using the `/docs` folder and [mkdocs](https://www.mkdocs.org/) with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/). Ensure you have the python 3.8 or newer installed and run `make` to install the dependencies and start the site.
 
 ### Python
-A core component of electionguard implemented in python which includes ballot encryption, decryption, key generation, and tallying.
 
-[📁 Source](https://github.com/microsoft/electionguard-python) | 
-[📦 Package](https://pypi.org/project/electionguard/) | 
+A core component of ElectionGuard implemented in python which includes ballot encryption, decryption, key generation, and tallying.
+
+[📁 Source](https://github.com/microsoft/electionguard-python) |
+[📦 Package](https://pypi.org/project/electionguard/) |
 [📝 Documentation](https://microsoft.github.io/electionguard-python/)
 
 ### C ++
+
 A subset of the ElectionGuard SDK implemented in C++ to support ballot encryption.
 
-[📁 Source](https://github.com/microsoft/electionguard-cpp)
+[📁 Source](https://github.com/microsoft/electionguard-cpp) |
+[📦 Package](https://www.nuget.org/packages/ElectionGuard.Encryption/) |
+[📝 Documentation](https://github.com/microsoft/electionguard-cpp#readme)
 
 ### Web API

```

Comment:
i know this was here before but maybe change this to `Python Fast API` or something tech-specific?  Web API was accurate before since that's the name of the .net product we were using, however now the meaning is generic.  It would be akin to labelling the C++ header in this section as "Encryption Package" or something and the Python header in this section being renamed to "Reference Implementation" or something like that.  thoughts?